src/localStorage.js

```diff
@@ -0,0 +1,14 @@
+export const localstorageStoreWelcomeBox = (welcomeBox) => {
+  localStorage.setItem('welcomeBox', welcomeBox);
```

Comment:
Strengt tatt burde vi sjekket for om dette går greit. Muligens er det enkleste bare å sette en "try-catch" rundt. Gjør gjerne noen søk på nettet. Men ihvertfall her tror jeg det skulle være greit (og dersom det feiler, kan du skrive en console.error, men ellers bare skippe lagring til localStorage, og returnere verdien slik du gjør nå.)